extension/mediator.js

```diff
@@ -43,6 +32,20 @@ class Mediator {
     }
 
     init() {
+        Sentry.init({
```

Comment:
Can't you only init when consent has been granted, so that we know that Sentry is disabled for sure?

Reply:
This will need to change and be refactored based on the outcome of: https://github.com/mozilla/firefox-translations/pull/269